src/common/Utils.ts

```diff
@@ -131,4 +133,22 @@ export function getUserAgent(): string {
         .replace("{product}", EXTENSION_NAME)
         .replace("{product-version}", getExtensionVersion())
         .replace("{comment}", "(" + getExtensionType()+'; )');
-}
\ No newline at end of file
+}
+
+export async function createAuthProfileExp(pacWrapper: PacWrapper | undefined) {
```

Comment:
this seems to be a good candidate for this util file - but beware that i recently added another copilotUtil file and we should use for copilot specific content.